tests/suites/ck/ck.sh

```diff
@@ -10,6 +10,9 @@ run_deploy_ck() {
 
 	ensure "${model_name}" "${file}"
 
+	if [[ $BOOTSTRAP_PROVIDER == "ec2" ]]; then
+		BOOTSTRAP_PROVIDER="aws"
+	fi
```

Comment:
Move this to the section where "ec2" is assigned, ie the reuse controller arg parsing

Reply:
donezo